internal/resource/interfaces.go

```diff
@@ -22,8 +22,8 @@ type ModelConfigService interface {
 	ModelConfig(context.Context) (*config.Config, error)
 }
 
-// Resources represents the methods used by the resource opener from state.Resources.
-type Resources interface {
+// ResourcesState represents the methods used by the resource opener from state.Resources.
+type ResourcesState interface {
```

Comment:
comment: I do not see this name change as an improvement. It feels like a regression on the path to better things.

Reply:
I've changed it to `DeprecatedResourcesState` to match the changes in https://github.com/juju/juju/pull/18510.